### DISCUSSION TITLE
make table of contents 'on by default' global preference option

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#7492] [Ideas]
Please add a global preference setting for admins to be able to enable showing the table of contents for all newly logged in users by default.
Users would still be able to turn off the table of contents as a per user setting, but this option would make the TOC enabled by default for all users.

Currently, showing the TOC by default is disabled for all users (with no ability to change this) and each user has to turn it on.

**Comment by ghost** [comment]
any traction on this?
Why do 'feature requests' get dumped into discussions?

I see enhancements being submitted in the 'issues' section, but someone has to cheat and say something is a bug. Feature requests aren't bugs.

**Reply by tommoor** [reply]
Because there are many feature requests and we try and keep issues focused on work that has been accepted.

In regards to this feature request, we'll consider it but we're trying to avoid adding more settings in general. It might be that the route forward here is to enable the TOC for everyone by default.

**Reply by ghost** [reply]
> we're trying to avoid adding more settings in general

why on Earth would you want to do that?
I support turning the TOC on by default though.

BUT, for every action you do by default, you should really allow options to the user to manipulate those defaults. You're never going to make everyone happy with whatever the default is, so if you at least allow the option for the user to adjust those defaults you will cut down on 'who decided this way is the best way'

**Comment by ghost** [comment]
so if a FR is accepted is it moved into the 'issues' section and tracked correctly? Or are only the FR's where people mis-represent them as bug reports show up there?

Is this FR accepted? rejected? I can't tell and leaving users in limbo isn't the right answer.

**Reply by tommoor** [reply]
Is this mainly a concern for publicly shared documents or internal users? Are you concerned that they will not be able to figure out how to enable the contents?

**Reply by ghost** [reply]
internal users. The ToC is a pretty critical component that isn't immediately obvious how to enable.
Because it's not immediately obvious, we would prefer it be enabled by default for all new users.

But the things is, my preference isn't what someone else's preference might be. So why not just make it a global option. Then it doesn't matter and you can make 1 change and make both camps happy.

**Comment by ghost** [comment]
was this ever turned into an actual feature request. This is something we'd like to see.
